cmds/bmp/bms_command.go

```diff
@@ -1,9 +1,19 @@
 package bmp
 
 import (
+	"errors"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+	"io/ioutil"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
 	clients "github.com/cloudfoundry-community/bosh-softlayer-tools/clients"
 	cmds "github.com/cloudfoundry-community/bosh-softlayer-tools/cmds"
 	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	tablewriter "github.com/olekukonko/tablewriter"
```

Comment:
DK also has a table writer in golang. Might want to ask him. I don't know this one so before we commit ask @cppforlife for his, maybe he knows how it compares to this one...